rest/handler.go

```diff
@@ -79,8 +79,8 @@ type handler struct {
 type handlerPrivs int
 
 const (
-	regularPrivs = iota // Handler requires authentication
-	publicPrivs         // Handler checks auth but doesn't require it
+	regularPrivs = iota // Handler requires valid authentication
+	publicPrivs         // Handler ensures valid auth if present, but doesn't require it
```

Comment:
this is not the case any more after the last update 😄 

```suggestion
	publicPrivs         // Handler checks auth and falls back to guest if invalid or missing
```

Reply:
Ah yes.. Haha